printing/pptactic.ml

```diff
@@ -1200,6 +1202,8 @@ module Make
             pr_may_eval pr.pr_constr pr.pr_lconstr pr.pr_constant pr.pr_pattern c
           | TacFreshId l ->
             keyword "fresh" ++ pr_fresh_ids l
+          | TacExactId x ->
+            primitive "exact_id" ++ pr_id x
```

Comment:
Shouldn't this be "ident_of_string"?  Or is this not part of the printing of Info and LtacProf?